docs/building_fledge/building_fledge.rst

```diff
@@ -73,6 +73,7 @@ Fledge is currently based on C/C++ and Python code. The packages needed to build
 - automake
 - avahi-daemon
 - build-essential
+- ca-certificates
```

Comment:
Would you mind to update the below console block as with

```
$ sudo apt-get install avahi-daemon curl git cmake g++ make build-essential ca-certificates autoconf automake

```

Reply:
log prints sample! Okay. seems we listed in alphabetical order and install order sample is fine in any order.